services/voting-node/README.md

```diff
@@ -0,0 +1,59 @@
+# General
+
+## Environment variables required by all nodes
+
+I use environment variables to configure my settings. These variables are the default values for my
+command-line options:
+
+* `EVENTDB_URL` - The URL of the event database
+
+#### Optional
+* `IS_NODE_RELOADABLE` - If set, the voting node will reload its configuration (optional). Defaults to `true`
+* `VOTING_HOST` - Voting node IP (optional). Defaults to `0.0.0.0`
+* `VOTING_PORT` - Voting node port (optional). Defaults to `8000`
+* `VOTING_LOG_LEVEL` - Log level (optional). Defaults to `info`
+* `VOTING_LOG_FORMAT` - Log format (optional). Defaults to `text`
+* `VOTING_NODE_STORAGE` - Path to node storage (optional). Defaults to `./node_storage`
+* `JORM_PATH` - Path to jormungandr executable (optional). Defaults to `jormungandr`
+* `JCLI_PATH` - Path to jcli executable (optional). Defaults to `jcli`
+
+## Environment variables required by `LEADER0` nodes
+
+### Secret Generation
+* `COMMITTEE_CRS` - The CRS is used to generate committee members, this is only used by leader0
+* `SECRET_SECRET` - The password used to encrypt/decrypt secrets in the database
```

Comment:
Typo `SECRET_PASSWORD`?

Reply:
Not a typo, it was named like this a while back, and is used as such in deployment. Shouldn't be a problem to change it if required.